index.html

```diff
@@ -59,60 +90,210 @@ <h2 id="overview">Overview</h2>
       cohesive resource.
     </p>
 
+    <h3>Who are we?</h3>
+
     <p>
-      We are the Integration and Coordination Center (ICC), a center within the
-      CFDE responsible for, among other things, collecting metrics associated
-      with Common Fund projects and providing them to the NIH. We do this in
-      close collaboration with other Centers and NIH program staff.
+      <a href="#contact">The authors of this page</a> are the Integration and
+      Coordination Center (ICC), a center within the CFDE.
     </p>
 
     <p>
-      This page contains instructions on how to
-      <strong>submit your Common Fund project</strong>
-      so that we can properly collect the required metrics on it.
+      We are responsible for (among other things) gathering info associated with
+      Common Fund projects and providing it to the NIH. We also aim to make
+      already-public info more accessible by presenting it in a centralized way.
     </p>
 
-    <h2 id="contact">Contact</h2>
+    <p>
+      We do not operate alone; we do this in close collaboration with other
+      centers and the NIH.
```

Comment:
NIH is very touchy about endorsing content. 
```suggestion
      CFDE centers.
```

Reply:
I've also pulled back mention of NIH in a few other spots now.